.catalog/changelog.md

```diff
@@ -1,2 +1,8 @@
 ## 0.1
  - Initial release
+
+## 0.1.1
```

Comment:
Given it hasn't been merged yet, you don't actually _need_ to do this, but if you do want to:
1. can you change to "0.2"?
2. can you order it with newest first (0.2 above 0.1)?